src/js/components/FormField/stories/ContentProps.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
```

Comment:
Can you please rename the file to Themed.js, it will be more straight forward to scale it in the future in case we'd like to demonstrate more theme changes of Form.

Reply:
Done